langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -82,6 +102,14 @@ public isolated function 'map(Type[] arr, @isolatedParam function(Type val) retu
 #
 # The parameter `func` is applied to each member of parameter `arr` in order.
 #
+# ```
+# int total = 0;
+# [1, 2, 3].forEach(function (int i) {
+#     total += i * 2;
```

Comment:
I think it is more simple if we use `total += i`. It will also match with the variable name `total`